physics/ephemeris_body.hpp

```diff
@@ -29,7 +29,7 @@ using ::std::placeholders::_3;
 
 namespace physics {
 
-namespace {
+namespace {  // TODO(egg): this should be a named namespace (internal)
```

Comment:
Could you create an issue instead of the TODO?  This is not the only place where we have this situation and I remain unconvinced that a namespace internal would help much.  (I'd prefer a namespace named after the file name.)